CHANGES.md

```diff
@@ -0,0 +1,79 @@
+# API Surface Changes
+
+This file describes API surface changes made during experimentation.
+A first Origin Trial was carried out between [M92–M94](https://chromestatus.com/feature/5597608644968448).
+No real feedback was shared by Google with other parties, incl. Intel.
+At the beginning of 2022, the ownership of the Compute Pressure API was transferred to Intel.
+
+## Changes since last Origin Trial
+
+After research and discussions with interested parties (e.g., Google, Zoom and others), we
+decided to make the following major changes to the API shape:
+
+- For better ergonomics, and due to security and fingerprinting [concerns](https://github.com/w3c/compute-pressure/issues/24), the API [interfaces](https://www.w3.org/TR/compute-pressure/#the-pressurerecord-interface) has been redesigned.
```

Comment:
has been redesigned to not use developer-configurable buckets but instead output pressure changes as high level (link to https://github.com/w3c/compute-pressure/blob/main/high-level-states.md)

Reply:
Fixed